tests/kernel/gen_isr_table/src/main.c

```diff
@@ -182,6 +182,23 @@ static int check_sw_isr(void *isr, u32_t arg, int offset)
 }
 #endif
 
+/** @brief test to validate gen_isr_table
+ *
```

Comment:
I think we should have  "@addtogroup kernel_interrupt_tests".

Reply:
@ingroup rather.